drv/gimlet-hf-server/src/main.rs

```diff
@@ -221,7 +221,7 @@ impl idl::InOrderHostFlashImpl for ServerImpl {
 
         set_and_check_write_enable(&self.qspi)?;
         self.qspi.page_program(addr, &self.block[..data.len()]);
-        poll_for_write_complete(&self.qspi);
+        poll_for_write_complete(&self.qspi, None);
```

Comment:
This leaves page_program as a busywait; given my measurements saw this come back as 0 or 1 tick to complete, I _think_ that's the right decision, but I could easily be convinced to switch this to `Some(1)` if that makes anyone nervous.